packages/graphback-core/src/runtime/CRUDService.ts

```diff
@@ -38,6 +39,10 @@ export class CRUDService<Type = any> implements GraphbackCRUDService<Type>  {
   }
 
   public async create(data: Type, context: GraphbackContext): Promise<Type> {
+    if (this.crudOptions.subCreate) {
```

Comment:
@wtrocki selecting all fields if subscription for mutation type is enabled.

Reply:
I think this will solve https://github.com/aerogear/graphback/issues/1684 right?